projects/node.js/project.yaml

```diff
@@ -0,0 +1,2 @@
+homepage: "https://nodejs.org"
+primary_contact: "security@nodejs.org"
```

Comment:
note that while this mailing list will get email notifications, we require individual email addresses associated with a google account to access crash testcases.